src/Test/FakeTemplating.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Sonata\BlockBundle\Test;
 
-use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Sonata\BlockBundle\Templating\EngineInterface;
```

Comment:
This is the only “use” that you are replackng even though there are more referencing to EngineInterface on the project. Dont we need to replace all of them?

Reply:
We will replace all "use"s in master branch, if this PR will be accepted. Tests, that use `Sonata\BlockBundle\Test\FakeTemplating` show us, that when we use templating service, implementing `Sonata\BlockBundle\Templating\EngineInterface`, everything is fine

Reply:
And we use `sonata.templating` instead of `templating` !

`sonata.templating` is `Sonata\BlockBundle\Templating\TwigEngine`, it implements `Sonata\BlockBundle\Templating\EngineInterface`.